### PR TITLE
Fix docs generation integration test

### PR DESCRIPTION
**Pull request by VersusFacit**
Resolves bug to an integration test introduced by merge of [multiple unique ids](https://github.com/dbt-labs/dbt-core/pull/4618) for integration tests 

### Description

The test diffs a pair of configs, in which must be certain parameters. Changes to core have necessitated the addition of `unique_id` to two configs in this test.

#### Note:

Although this PR is small, it represents the same contextual work done with the matching [Snowflake fix](https://github.com/dbt-labs/dbt-snowflake/pull/94). 

### Checklist

- [X] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [X] I have run this code in development and it appears to resolve the stated issue
- [X] This PR includes tests, or tests are not required/relevant for this PR
- [X] I have updated the `CHANGELOG.md` and added information about my change to the "dbt-snowflake next" section.

### CODE REVIEW

File: tests/integration/docs_generate_tests/test_docs_generate.py [hunk 1]
```diff
@@ -516,6 +518,7 @@ def rendered_snapshot_config(self, **updates):
             'unique_key': 'id',
             'target_schema': None,
             'meta': {},
+            'unique_key': 'id'
```
Comment (leahwicz) [inline tests/integration/docs_generate_tests/test_docs_generate.py:521]: Is this a duplicate of line 518?

Comment (VersusFacit) [reply tests/integration/docs_generate_tests/test_docs_generate.py]: Shoot! An artifact of debugging and trying to parse those darn diffs. Didn't realize it would pass duplicates. Thanks for noticing.

**Review by leahwicz — APPROVED** [review]

**Comment by VersusFacit** [comment]
Thanks for the pointer. Got Redshift tests happy again with a touch of debugging following the read through of the Core PR! I'll be sure to follow that process in the future should "random" bugs arise

File: tests/unit/test_context.py [hunk 2]
```diff
@@ -183,6 +183,7 @@ def assert_has_keys(
     'invocation_id',
     'modules',
     'flags',
+    'print'
```
Comment (leahwicz) [inline tests/unit/test_context.py:186]: Was this addition needed b/c of this PR https://github.com/dbt-labs/dbt-core/pull/4701?

Comment (leahwicz) [reply tests/unit/test_context.py:186]: If so, I'm guessing the other adapters might need this update as well

Comment (VersusFacit) [reply tests/unit/test_context.py:186]: It was. I'll merge this PR and look at the others

Comment (VersusFacit) [reply tests/unit/test_context.py:186]: Doubled checked snowflake, bigquery, and spark. The unit tests of those are passing just fine! No additional fixes needed for this issue looks like. 😄
